int32 type
int32 POSTURE = 1
int32 GESTURE = 2
int32 BOTH = 3

bool merge
primitive_msgs/PoseTrajectoryPoint posture
GestureCommand gesture
